src/plugins/relatedContentPlugin.tsx

```diff
@@ -94,7 +94,7 @@ const getRelatedArticleProps = (
       )) ||
     article.resource.path;
 
-  let to = path;
+  let to = path || '';
```

Comment:
Kan det være denne som gjør at tabindex er borte? No kan ikkje lenger to være undefined.

Reply:
Der var løsningen ja 🥳

Reply:
Men det er kanskje litt rart at dette fjerna tabindex?